File: server/src/Tables/Player.cs

using SpacetimeDB;

namespace masks.server.Tables;

[Table(Name = "Player", Public = true)]
[Table(Name = "LoggedOutPlayer")]
public partial struct Player
{
    [PrimaryKey] 
    public Identity Identity;
    [Unique, AutoInc] 
    public uint Id;
    public string Name;
    public bool IsPaused;
}

File: server/src/tables/player.rs
use spacetimedb::Identity;

/// Table: Player
#[spacetimedb::table(name = player, public)]
#[spacetimedb::table(name = logged_out_player)]
#[derive(Debug, Clone)]
pub struct Player {
    #[primary_key]
    pub identity: Identity,
    #[unique]
    #[auto_inc]
    pub id: u32,
    pub username: String,
    pub is_paused: bool,
}